tests/Utilities/testJobDefinitions.py

```diff
@@ -77,7 +77,24 @@ def mpJob():
                       [find_all('testMpJob.sh', os.environ['WORKSPACE'], 'DIRAC/tests/Utilities')[0]])
 
   J.setExecutable('testMpJob.sh mpTest.py')
-  J.setTag('MultiProcessor')
+  J.setTag(['4Processors', 'MultiProcessor'])
```

Comment:
```suggestion
  J.setNumberOfProcessors(4)
```

Reply:
Nah, leave this one. I have done it on purpose to be compatible with older version.